content/blocks.py

```diff
@@ -0,0 +1,5 @@
+from wagtail.core.blocks import RichTextBlock
+
+class ContentBlock(RichTextBlock):
+    class Meta:
+        template = 'content/content_block.html'
```

Comment:
We don't enforce code style yet, but is it a good practice to end files with a new line.